components/productImage/leftarrow/leftarrow.js

```diff
@@ -0,0 +1,18 @@
+import  { Component } from "react";
+import React from "react";
```

Comment:
All this arrow does is render a arrow or button . Why is it a different component ? Make it a part of carusal